app/models/line_item.rb

```diff
@@ -0,0 +1,16 @@
+class LineItem < ApplicationRecord
+  belongs_to :deal
+  belongs_to :order
+
+  def unit_price
+    discount_price_in_cents + tax_on_deal
+  end
+
+  def total_price
+    unit_price * quantity 
+  end
+
+  def tax_on_deal
+    deal.deals_tax  * discount_price_in_cents * 0.01
```

Comment:
discount_price_in_cents * 0.01 -> this should be a method called discount_price
All _in_cents columns should have a corresponding method
Ex - discount_price_in_cents should have a method called discount_price

Reply:
done